serving/ingress/deploy/networkpolicies.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: knative-cluster-local-gateway-hack
+  namespace: knative-serving-ingress
```

Comment:
I would prefer not to hard-code this ns. I think the right place to install this is in the new operator, where we create this ns and install the servicemesh.
/hold